### PR TITLE
Updating README commands

### PR DESCRIPTION
**Pull request by racheljpg**
I followed the README for the first time and ran into some problems, so this is a PR to update some commands and ask some questions :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Java, version 11 or newer, and Apache Maven are required for building this proje
 Build the project like this:
 
 ```
-$ mvn clean verify
+$ mvn clean install
```
Comment (racheljpg) [inline README.md:12]: When I ran `mvn clean verify` it wouldn't work as it told me I didn't have a specific (kroxylicious) related component installed. I'm afraid I forgot to note exactly what one it was, though - but switching this to install worked for me.

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ $ mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@resolv
 Build with the `dist` profile as shown above, then execute this:
 
 ```
-$ java -jar kroxylicious/target/kproxy-1.0-SNAPSHOT.jar
+$ java -jar kroxylicious/target/kroxylicious-1.0-SNAPSHOT.jar
```
Comment (racheljpg) [inline README.md:62]: The name seemed to be different for me compared to the README, so I'm updating it.
I cannot actually run this though, as I get this error:
`java -jar kroxylicious/target/kroxylicious-1.0-SNAPSHOT.jar 
Missing required option: '--config=<configFile>'`
What config file is this? And how do I create one, so I can add this step in here also?

File: kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java [hunk 3]
```diff
@@ -62,7 +62,7 @@ public void initChannel(SocketChannel ch) {
         if (logFrames) {
             pipeline.addLast("frameLogger", new LoggingHandler("io.kroxylicious.proxy.internal.DownstreamFrameLogger", LogLevel.INFO));
         }
-        
+
```
Comment (racheljpg) [inline kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java:65]: It seems that the `mvn process-sources` command is removing trailing spaces for us. Is this something that perhaps could be added to the GitHub action that runs on each of these PRs?

File: README.md (hunk 1 above)
Comment (SamBarker) [reply README.md:12]: hmm `verify` should be enough, it loosely implies we've not got the module relationships right in maven. Having said that I've no problems with having `install` in the readme.

Comment (gunnarmorling) [reply README.md:12]: Yeah, I was wondering about this too. Part of this aggregator build is a plug-in which then is applied to the actual proxy module. When not installing the former, a build of only the latter will fail. So all in all, `install` should be the more robust choice and it makes sense to have it here.

**Review by grdryn — COMMENTED** [review]
@racheljpg FYI this needs to be rebased locally and re-pushed, as there are conflicts with the main branch.

**Comment by shivam-sharma7** [comment]
Hey @SamBarker / I want to contrubute in this project. I have intermediate knowledge in java. How can i start with this project? what should i learn to contribute in this project.

**Comment by prakrit55** [comment]
hey @racheljpg  its not working with install as well , its saying krpc plugin not defined, and tests are not able to complete.

File: README.md (hunk 1 above)
Comment (SamBarker) [reply README.md:12]: I've raised https://github.com/kroxylicious/kroxylicious/issues/106 to cover fixing `verify`.

**Review by SamBarker — APPROVED** [review]

**Comment by SamBarker** [comment]
Thanks for the PR @racheljpg.

@prakrit55 can you raise an issue with the full build log if you are still having issues after trying a JDK 17+ build?

**Comment by prakrit55** [comment]
> hey @prakrit55, what version of Java are you using? If it's not 17+ you can try updating that and giving it another try. What's the exact output/errors you get?

I had been running it in java -v 17, I think it will run in higher than that.

About the errors, its asking about couldnt build krpc plugin and had stuck there.

https://github.com/kroxylicious/kroxylicious/issues/102#issuecomment-1372561668

**Comment by prakrit55** [comment]
> Thanks for the PR @racheljpg.
> 
> @prakrit55 can you raise an issue with the full build log if you are still having issues after trying a JDK 17+ build?

hey @SamBarker  let me try in higher version of java , then I will let you know.

**Pull request merged**